AddTestGuide.md

```diff
@@ -0,0 +1,197 @@
+# USF OpenTripPlanner Adding Testing cases instructions
+
+This readme is intend to provide information solely about adding a new test case. For more information about how to use this test unit please refer to the main readme (It can be found here: https://github.com/CUTR-at-USF/test/tree/WIP-jmfield2#usf-opentripplanner-testing-framework).
+
+
+## The test cases can be found in the following spreadsheet:
+
+https://docs.google.com/spreadsheets/d/1f_CTDgQfey5mY1eMO03D7UZ8855D-mxHsfYfsA3c4Zw
+
+## Adding a test case
+
+Note: To add a new case the right to edit the spreadsheet is necessary (Link above).
+
+**Example:**
+
+```
+$ python test_runner.py --add "http://mobullity.forest.usf.edu/index.html?module=planner&fromPlace=28.06914026483356%2C-82.40278244018555&toPlace=28.069272801561006%2C-82.40248203277588&time=6%3A09pm&date=07-23-2015&mode=TRANSIT%2CWALK&maxWalkDistance=1609.34&wheelchair=false&arriveBy=false&bannedRoutes=undefined&showIntermediateStops=true&itinIndex=0" --add-class=USFPlanner -R
+```
+
+The script ```test_runner.py``` need to be run with the following attribut:
+
+```
+--add ""
+```
+
+The itinerary link retrieved from the planner need to be inside of the quote. 
+To copy the itinerary link from the planner, first plan your trip then in the bottom of the detail of this trip there should be a link named "Link to Itinerary", you only need to copy the url of this link.
+
+```
+--add-class=USFPlanner -R
```

Comment:
This is correct, but perhaps a mention that currently the only test class which supports the add test feature is the USFPlanner -- because it implements the 'valid_url_parameters' static method.